### ISSUE TITLE
Feature request: ping & message & reminder default tester/manager on stale test run

### ISSUE THREAD

**Issue by atodorov** [kiwitcms/Kiwi#717]
Sometimes we have test runs which are left incomplete for whatever reason: person forgot to update them, priorities shifted, etc. For example https://tcms.kiwitcms.org/runs/5/

These should be either completed (all statuses != IDLE) or set to Finished.  If this is not the case then we need 2 things:

1) A weekly reminder to tester assigned to IDLE case, TR default tester, and manager in CC to complete the task
2) A sort of ping button (with the possibility of adding a message) which will serve as the reminder as well (can be received via email too).  This can also possibly integrate with a framework which will allow sending messages to various users on the website.
3) These pings & notifications must be reported somewhere, possibly TR history.

Note: split into multiple issues when things start to become big.

CC @renystankova

**Comment by Prome88** [comment]
Maybe an integration / some kind of bot for popular IM programs such as Slack?

**Comment by atodorov** [comment]
> Maybe an integration / some kind of bot for popular IM programs such as Slack?

Good idea. We can implement the interface and then the emailing backend will be the POC. Next Slack or other bots can be added as desired.

**Comment by rbenzair** [comment]
+1
